Skyflow/src/main/kotlin/Skyflow/TextField.kt

```diff
@@ -261,8 +261,13 @@ class TextField @JvmOverloads constructor(
     }
     private fun changeCardIcon(cardtype:CardType)
     {
-            inputField.setCompoundDrawablesRelativeWithIntrinsicBounds(cardtype.image, 0, 0, 0)
-            inputField.compoundDrawablePadding = 8
+            if(!cardtype.equals(CardType.EMPTY)) {
```

Comment:
indentation

Reply:
please maintain consistency for card Types. js is using UNKNOWN card type. Here we are using EMPTY